__init__.py

```diff
@@ -1,4 +0,0 @@
-from rooms import Room, Office, LivingSpace
```

Comment:
**Code Quality - Nitpick:** It's good you deleted this, it's best practice not to have code inside `__init__.py` file.s